src/domains/Restaurants/ManageRestaurants.tsx

```diff
@@ -17,6 +22,7 @@ export const ManageRestaurants = () => {
   const { user } = React.useContext(UserContext);
 
   const restaurant = watch();
+  console.log(restaurant);
```

Comment:
Just to get in the habit, in prod things shouldn't have console logs - could you remove this?